### PR TITLE
Homework for Python

### PR DESCRIPTION
**Pull request by haozhao2002**
老师这次应该成功了。搞了半天才明白需要git switch homework，好崩溃。虽然尝试了无数次，不过还好总算成功了，学到了很多经验！

### CODE REVIEW

**Comment by iphysresearch** [comment]
Good Job!!!   @haozhao2002
---
Python homework:
- Total questions: 108
- Correct answers: 108
- Score: 100.00%
---
Numpy homework:
- Total questions: 10
- Correct answers: 10
- Score: 100.00%
---
Pandas homework:
- Total questions: 12
- Correct answers: 12
- Score: 100.00%
---

**Comment by haozhao2002** [comment]
leetcode162：
leetcode162
leetcode434：
leetcode434
leetcode852：
leetcode852
leetcode1784：
leetcode1784
leetcode1869：
leetcode1869

**Comment by haozhao2002** [comment]
> ## Good Job!!! @haozhao2002
> Python homework:
> 
> * Total questions: 108
> * Correct answers: 108
> * Score: 100.00%
> 
> Numpy homework:
> 
> * Total questions: 10
> * Correct answers: 10
> * Score: 100.00%
> 
> Pandas homework:
> 
> * Total questions: 12
> * Correct answers: 12
> * Score: 100.00%

这个机器人好智能，竟然能自动打分！

File: 2023/homework/Zhao_Hao/homework-matplotlib.ipynb
Comment (iphysresearch) [inline 2023/homework/Zhao_Hao/homework-matplotlib.ipynb:1]: Matplotlib作业！

**Review by iphysresearch — APPROVED** [review]
@haozhao2002 
恭喜完成Python基础+扩展作业，Matplotlib数据可视化作业！

**Pull request merged**